snippets/curry.md

```diff
@@ -5,8 +5,15 @@ If the number of provided arguments (`args`) is sufficient, call the passed func
 Otherwise return a curried function `f` that expects the rest of the arguments.
 
```

Comment:
Still waiting on the updated description, apart from that the function is solid. Thanks for contributing yet again! 👍